src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -850,7 +850,7 @@ describe('Add answer mappings to concept', () => {
     moxios.uninstall();
   });
 
-  it('should add all chosen answer mappings', async () => {
```

Comment:
Is this related to the react scripts version change?

Reply:
Yes, the test fails if it's wrapped in an async..await block

Reply:
Has that got anything to do with the react scripts version change?

Reply:
it fails when using react-scripts >=2.0.0